src/shared/sleep-products.conf

```diff
@@ -33,4 +33,6 @@
 #
 #  Note: All the sections and variables are optional
 
+[CanSuspend]
```

Comment:
I assume you'd like to add the product names to CanHibernate and CanHybridSleep too, just in case.

Reply:
Those are disabled for Endless